Loruki Clone/index.html

```diff
@@ -0,0 +1,375 @@
+          <h2>Request a Demo</h2>
+          <form onsubmit="event.preventDefault()">
+            <div class="form-control">
+              <input type="text" name="name" placeholder="Name" required />
+            </div>
+            <div class="form-control">
+              <input
+                type="text"
+                name="company"
+                placeholder="Company Name"
+                required
+              />
+            </div>
+            <div class="form-control">
+              <input type="email" name="email" placeholder="Email" required />
+            </div>
+            <input type="submit" value="Send" class="btn btn-primary" />
+          </form>
+        </div>
+      </div>
+    </section>
+
+    <!-- stats -->
+    <section class="stats">
+      <div class="container">
+        <h3 class="stats-heading text-center my-1">
+          Welcome to the best platform for building applications of all types
+          with modern architecture and scaling
+        </h3>
+        <div class="grid grid-3 text-center my-4">
+          <div>
+            <i class="fas fa-server fa-3x"></i>
+            <h3>10,349,405</h3>
+            <p class="text-secondary">Deployments</p>
+          </div>
+          <div>
+            <i class="fas fa-upload fa-3x"></i>
+            <h3>987 TB</h3>
+            <p class="text-secondary">Published</p>
+          </div>
+          <div>
+            <i class="fas fa-project-diagram fa-3x"></i>
+            <h3>2,343,265</h3>
+            <p class="text-secondary">Projects</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- cli -->
+    <section class="cli">
+      <div class="container grid">
+        <img
+          src="https://github.com/bradtraversy/loruki-website/blob/master/images/cli.png?raw=true"
+          alt=""
+        />
+        <div class="card">
+          <h3>Easy to use, cross platform CLI</h3>
+        </div>
+        <div class="card">
+          <h3>Deploy in seconds</h3>
+        </div>
+      </div>
+    </section>
+
+    <!-- cloud -->
+    <section class="cloud my-2 py-2">
+      <div class="container grid bg-secondary">
+        <div class="text-center">
+          <h2 class="lg">Extreme Cloud Hosting</h2>
+          <p class="lead my-1">
+            Cloud hosting like you've never seen. Fast, efficient and scalable
+          </p>
+          <a href="#features" class="btn btn-dark">Read More</a>
+        </div>
+        <img
+          src="https://github.com/bradtraversy/loruki-website/blob/master/images/cloud.png?raw=true"
+          alt=""
+        />
+      </div>
+    </section>
+
+    <!-- languages -->
+    <section class="languages">
+      <h2 class="md text-center my-2">Supported Languages</h2>
+      <div class="container flex">
+        <div class="card">
+          <h4>Node.js</h4>
+          <img
+            src="https://github.com/bradtraversy/loruki-website/blob/master/images/logos/node.png?raw=true"
```

Comment:
Code me changes hai ya same utha liye? @aaryahjolia

Reply:
Ye to referenced website hi daal hi galti se. Ruko Naya pr banata hoon 2 din me abhi out of station hoon